libraries/AP_HAL_SITL/SITL_cmdline.cpp

```diff
@@ -42,22 +42,28 @@ static void _sig_fpe(int signum)
 void SITL_State::_usage(void)
 {
     printf("Options:\n"
-           "\t--home HOME        set home location (lat,lng,alt,yaw)\n"
-           "\t--model MODEL      set simulation model\n"
-           "\t--wipe             wipe eeprom and dataflash\n"
-           "\t--unhide-groups    parameter enumeration ignores AP_PARAM_FLAG_ENABLE\n"
-           "\t--rate RATE        set SITL framerate\n"
-           "\t--console          use console instead of TCP ports\n"
-           "\t--instance N       set instance of SITL (adds 10*instance to all port numbers)\n"
-           "\t--speedup SPEEDUP  set simulation speedup\n"
-           "\t--gimbal           enable simulated MAVLink gimbal\n"
-           "\t--autotest-dir DIR set directory for additional files\n"
-           "\t--uartA device     set device string for UARTA\n"
-           "\t--uartB device     set device string for UARTB\n"
-           "\t--uartC device     set device string for UARTC\n"
-           "\t--uartD device     set device string for UARTD\n"
-           "\t--uartE device     set device string for UARTE\n"
-           "\t--defaults path    set path to defaults file\n"
+           "\t--home HOME              set home location (lat,lng,alt,yaw)\n"
+           "\t--model MODEL            set simulation model\n"
+           "\t--wipe                   wipe eeprom and dataflash\n"
+           "\t--unhide-groups          parameter enumeration ignores AP_PARAM_FLAG_ENABLE\n"
+           "\t--rate RATE              set SITL framerate\n"
+           "\t--console                use console instead of TCP ports\n"
+           "\t--instance N             set instance of SITL (adds 10*instance to all port numbers)\n"
+           "\t--speedup SPEEDUP        set simulation speedup\n"
+           "\t--gimbal                 enable simulated MAVLink gimbal\n"
+           "\t--autotest-dir DIR       set directory for additional files\n"
+           "\t--uartA device           set device string for UARTA\n"
+           "\t--uartB device           set device string for UARTB\n"
+           "\t--uartC device           set device string for UARTC\n"
+           "\t--uartD device           set device string for UARTD\n"
+           "\t--uartE device           set device string for UARTE\n"
+           "\t--gazebo-address ADDR    set address string for gazebo\n"
+           "\t--gazebo-port-in PORT    set port num for gazebo in\n"
+           "\t--gazebo-port-out PORT   set port num for gazebo out\n"
+           "\t--irlock-port PORT       set port num for irlock\n"
```

Comment:
Only Gazebo use it for now but theorically anything can use it as you just need to open the socket and send the irlock_packet !